modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2805,6 +2805,28 @@ public void testGetValuesList() throws Exception {
 		_assertObjectEntryValues(29, values1, valuesList.get(0));
 		_assertObjectEntryValues(29, values3, valuesList.get(1));
 
+		// Predicate and Search
+
+		String search = "John";
```

Comment:
We don't need this new variable.
Can you inline the usages?

In the first call, I would put "John" directly.
In the second call, I would put `RandomTestUtil.randomString()`.

Okay?